<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>417</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>626</width>
    <height>417</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>626</width>
    <height>417</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Kodchasan</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Player</string>
  </property>
  <widget class="QWidget" name="centralwidget_players">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="widget_fondo" native="true">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>361</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>281</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Player 1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="QlineEdit_player_1_nick">
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
        </property>
        <property name="placeholderText">
         <string>Nick Name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Player 2</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="QlineEdit_player_2_nick">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
        </property>
        <property name="placeholderText">
         <string>Nick Name</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="QpushButton_jugar">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>260</y>
       <width>281</width>
       <height>61</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>281</width>
       <height>61</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>281</width>
       <height>61</height>
      </size>
     </property>
     <property name="text">
      <string>Jugar</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
